frontend-rsnt/src/components/DesktopOnlyCockBlock.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {};
+
+export const DesktopOnlyCockBlock: React.FC<Props> = (props) => (
```

Comment:
Please, rename the this variable name. I understand the purpose of this function, but I am sure that there are better ways to achieve this.